cmd/aws/main.go

```diff
@@ -49,11 +49,26 @@ var (
 	notAfterStart timestampFlag
 	notAfterLimit timestampFlag
 
-	httpEndpoint               = flag.String("http_endpoint", "localhost:6962", "Endpoint for HTTP (host:port).")
-	httpDeadline               = flag.Duration("http_deadline", time.Second*10, "Deadline for HTTP requests.")
-	maskInternalErrors         = flag.Bool("mask_internal_errors", false, "Don't return error strings with Internal Server Error HTTP responses.")
-	origin                     = flag.String("origin", "", "Origin of the log, for checkpoints and the monitoring prefix.")
-	bucket                     = flag.String("bucket", "", "Name of the bucket to store the log in.")
+	// Functionality flags
+	httpEndpoint       = flag.String("http_endpoint", "localhost:6962", "Endpoint for HTTP (host:port).")
+	httpDeadline       = flag.Duration("http_deadline", time.Second*10, "Deadline for HTTP requests.")
```

Comment:
This one would be better located with performance flags I think

Reply:
Moved. We can rearrange if we find a better place later.